csharp/test/Ice/optional/TestI.cs

```diff
@@ -15,7 +15,6 @@ internal class Test : ITest
 
         public void OpSingleInString(string? i1, Current current) => TestHelper.Assert(i1 == null || i1 == "42");
 
-        public int? OpSingleOutInt(Current current) => 42;
         public string? OpSingleOutString(Current current) => "42";
```

Comment:
You should remove this line for `OpSingleOutString`, since you removed the corresponding operation from the Slice file. You can see it in the file above this one on line 62
You should also remove anywhere else where we might call this method in the test.

Reply:
You should still remove this `OpSingleOutString` function from C# since you removed it from the Slice file